intro/calc.js

```diff
@@ -0,0 +1,39 @@
+class Calculator {
+  static add(a, b) {
+    if (isNaN(a) || isNaN(b)) {
+      throw new Error("Both inputs must be numbers.");
+    }
+    return a + b;
+  }
+
+  static subtract(a, b) {
+    if (isNaN(a) || isNaN(b)) {
+      throw new Error("Both inputs must be numbers.");
+    }
+    return a - b;
+  }
+
+  static divide(a, b) {
+    if (isNaN(a) || isNaN(b)) {
+      throw new Error("Both inputs must be numbers.");
+    }
+    if (b === 0) {
+      throw new Error("Division by zero is not allowed.");
+    }
+    return a / b;
+  }
+
+  static multiply(a, b) {
+    if (isNaN(a) || isNaN(b)) {
+      throw new Error("Both inputs must be numbers.");
+    }
```

Comment:
Since these blocks of code, which validates parameters, repeated more than once, you can create another function to validate parameters, then call that function instead of `if` statement.